cpp/src/reductions/arg_minmax_util.cuh

```diff
@@ -28,17 +28,19 @@ namespace detail {
  * @tparam T Type of the underlying data. This is the fallback for the cases when T does not support
  * '<' operator.
  */
-template <bool has_nulls>
```

Comment:
Can you remove the `@tparam` line above, please? I forgot to do that when copy/past the code.